multipart/multipart.py

```diff
@@ -45,6 +45,13 @@
 STATE_PART_DATA_END             = 10
 STATE_END                       = 11
 
+STATES = [
```

Comment:
I found it useful for debugging.  I thought it might be useful in the future.   Let me know if you still want it remove.